docs/source/usage/quick_start.rst

```diff
@@ -0,0 +1,101 @@
+Quick start guide
+=================
+
+Get started quickly with MetaSynth using the following example. In this concise demonstration, you'll learn the basic functionality of MetaSynth by generating synthetic data from `titanic <https://raw.githubusercontent.com/pandas-dev/pandas/main/doc/data/titanic.csv>`_ dataset.
+
+.. note:: 
+   The steps on this page correspond to the basic tutorial available on the :doc:`/usage/interactive_tutorials` page. As such, if you prefer an interactive experience, check out the basic tutorial for a guided walkthrough!
+
+Importing Libraries
+-------------------
+
+The first step in any Python project is to import the necessary libraries. For this example, we will need Polars and MetaSynth.
+
+
+.. code:: python
+
+   import polars as pl
+   import metasynth as ms
+
+
+Loading the Dataset
+-------------------
+
+Next, load the Titanic CSV file, for this we can use the built-in :meth:`metasynth.demo_file` function.
+
+.. code-block:: python
+
+   dataset_csv = ms.demo_file() 
+
+Then, we create a dictionary with information on the data types of the various columns/variables. This will be used in the next step, when we convert the CSV file to a DataFrame.
+
+.. code-block:: python
+
+   data_types = { 
+       "Sex": pl.Categorical,
+       "Embarked": pl.Categorical,
+       "Survived": pl.Categorical,
+       "Pclass": pl.Categorical,
+       "SibSp": pl.Categorical,
+       "Parch": pl.Categorical
+   }
+
+To finish loading the dataset, we simply use the :meth:`polars.read_csv` function, passing in our CSV file and dictionary as parameters. 
+
+.. code-block:: python
+
+   dataframe = pl.read_csv(dataset_csv, dtypes=data_types)
+
+
+This converts the CSV file into a DataFrame.
+
+.. note:: 
+	In this example, we used a Polars DataFrame, but Pandas DataFrames are also supported by MetaSynth. 
+
+
+Generating the MetaDataset
+--------------------------
+With the DataFrame loaded, you can now generate a :obj:`MetaDataset<metasynth.dataset.MetaDataset>`.
+
+
+.. code-block:: python
+
+   metadataset = ms.MetaDataset.from_dataframe(dataframe)
```

Comment:
Also update the `from_dataframe` method.

Reply:
Same for this one, I'll do the renaming after the merge.